float64[14] goal
float64 v
---
